src/views/Rules/utils/deleteRule.jsx

```diff
@@ -0,0 +1,22 @@
+import tryCatch from '../../../utils/tryCatch';
+import { deleteRule } from '../../../utils/Rules';
+
+export default async rule => {
+  // TODO: why can't we index the result to just get the error?
+  // await stops working if we do
+  const result = await tryCatch(
+    deleteRule({
+      ruleId: rule.rule_id,
+      dataVersion: rule.data_version,
+    })
+  );
+
+  if (result[0]) {
+    const errorMsg =
+      result[0].response.data.exception ||
+      result[0].response.data.detail ||
+      'Unknown Error';
```

Comment:
Inserting `console.log('pairs: ', Object.entries(result[0]));` shows:

<img width="500" alt="Screen Shot 2019-06-16 at 4 14 45 PM" src="https://user-images.githubusercontent.com/3766511/59570756-e7f73780-9051-11e9-82bb-849641b4bf61.png">

which suggests the reason this is failing is because `response` is set to`null`.

<img width="157" alt="Screen Shot 2019-06-16 at 4 15 54 PM" src="https://user-images.githubusercontent.com/3766511/59570764-0b21e700-9052-11e9-91b6-1e873a02986d.png">

Reply:
I'm guessing that you got this error because the backend wasn't running or available? Either way it should be addressed, just wondering about the context.

Reply:
Maybe. It works now :)